definitions/npm/nock_v9.x.x/flow_v0.31.x-/nock_v9.x.x.js

```diff
@@ -38,13 +38,13 @@ declare type $npm$nock$InterceptorOptions = {
 };
 
 declare class $npm$nock$NockBack {
-  $call(path: string, cb: (cb: Function) => any): void,
+  $callable(path: string, cb: (cb: Function) => any): void,
```

Comment:
You'll need to cap this at `flow_v0.74`, and then copy this file over to a new folder `flow_v0.75.x-`. I think `$callable` was introduced in `0.75` yeah?

Reply:
Ah ok, that makes sense! I'll try it out.